### PR TITLE
Updates roadmap Q4 2023

### LINKED ISSUE #1338: Update roadmap.md
Currently references members - needs to be updated

### PR DESCRIPTION
**Pull request by clausmullie**
Closes #1338

### CODE REVIEW

File: organization/roadmap.md [hunk 1]
```diff
@@ -30,29 +31,27 @@ To this end we will expand our capabilities and capacity to continuously produce
...
-### Memberships
+### Chapters
 
-Our most active focus for membership in the Foundation will shift toward provinces, states, and national governments.
-If the Foundation for Public Code begins to prove value within a network of regional municipal partners, it makes most sense to take the case for structural support to the infrastructural agencies of the ‘parent’ state.
-We are moving conversations forward with national governments in Europe and North America, and expect to slowly scale the rate at which we add state-level members.
+The Foundation for Public Code is transforming its organizational model to enable international chapters.
+There will be a global umbrella organization, along with local chapter organizations.
+The umbrella organization will be an association in which the strategic council, board of directors, and delegates from the chapter organizations work together to define long term priorities.
+Local chapters will tailour their work based on the needs of the local public code ecosystem and participating public organizations.
```
Comment (ElenaFdR) [inline organization/roadmap.md:49]:
```suggestion
Local chapters will tailor their work based on the needs of the local public code ecosystem and participating public organizations.
```

File: organization/roadmap.md [hunk 2]
```diff
@@ -30,29 +31,27 @@ To this end we will expand our capabilities and capacity to continuously produce
...
+The North America chapter is the first chapter of the Foundation for Public Code.
```
Comment (ElenaFdR) [inline organization/roadmap.md:50]:
```suggestion
The North America chapter is the Foundation for Public Code's first local chapter.
```

File: organization/roadmap.md [hunk 3]
```diff
@@ -30,29 +31,27 @@ To this end we will expand our capabilities and capacity to continuously produce
...
 
-### Memberships
+### Chapters
 
-Our most active focus for membership in the Foundation will shift toward provinces, states, and national governments.
-If the Foundation for Public Code begins to prove value within a network of regional municipal partners, it makes most sense to take the case for structural support to the infrastructural agencies of the ‘parent’ state.
-We are moving conversations forward with national governments in Europe and North America, and expect to slowly scale the rate at which we add state-level members.
+The Foundation for Public Code is transforming its organizational model to enable international chapters.
+There will be a global umbrella organization, along with local chapter organizations.
+The umbrella organization will be an association in which the strategic council, board of directors, and delegates from the chapter organizations work together to define long term priorities.
```
Comment (ElenaFdR) [inline organization/roadmap.md:48]:
```suggestion
The global organization will be an association in which the strategic council, board of directors, and delegates from the chapter organizations work together to define long term priorities.
```

**Review by ElenaFdR — APPROVED** [review]
This is an excellent lightweight update to reflect our current reality as of Oct 2023, and will provide us with a good base for iterating the next time we/the board have a more in-depth roadmapping session.

**Review by nebster — APPROVED** [review]
this looks great.

**Pull request merged**